app/controllers/public/cookie_statement_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Public::CookieStatementController < ApplicationController
+  include Pundit
```

Comment:
I see we have a test that is checking we can see the cookie policy when both signed in and signed out which is great. It seems a bit odd to have authorisation code loaded into a controller with the name "public".

I think I've answered my own question: It looks to be because the `_navigation` partial that's rendered has logic that uses Pundit methods. This makes sense as we want the same view to be rendered for staff and the public without needing to duplicate views. 

Do you think it is worth including these in each public or API controller as we need or should we include Pundit in all controllers as a default?

Reply:
Assuming all pages will need the navigation maybe we should add it everywhere.